.travis.yml

```diff
@@ -1,5 +1,5 @@
 # trusty beta image has jdk8, gcc4.8.4
```

Comment:
```suggestion
```

Reply:
I'll merge this change in a moment; I am still fixing some minor errors in this.